exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -584,6 +584,8 @@ drill.exec.options: {
     planner.partitioner_sender_threads_factor: 2,
     planner.producer_consumer_queue_size: 10,
     planner.slice_target: 100000,
+    planner.statistics.parallel_analyze: true,
```

Comment:
I don't think we need to distinguish parallel/sequential analyze.  There should be only one version (parallel) just as with all other Drill operators.

Reply:
Done